components/raftstore/src/store/worker/pd.rs

```diff
@@ -1218,45 +1228,76 @@ where
                                 stats: stats_copy,
                                 store_info,
                                 send_detailed_report: true,
+                                forced_leaders: None,
                                 dr_autosync_status,
                             };
                             if let Err(e) = scheduler.schedule(task) {
                                 error!("notify pd failed"; "err" => ?e);
                             }
                         }
                     } else if resp.has_plan() {
```

Comment:
remove `get_require_detailed_report` and reuse `has_plan`, always send report after executing plan(maybe emtpy)